src/home_robot/home_robot/perception/detection/lseg/modules/lsegmentation_module.py

```diff
@@ -11,14 +11,15 @@
 
 # add mixed precision
 import torch.cuda.amp as amp
-from data import get_available_datasets, get_dataset
 from encoding.nn import SegmentationLosses
 from encoding.utils import (
     SegmentationMetric,
     batch_intersection_union,
     batch_pix_accuracy,
 )
 
+from data import get_available_datasets, get_dataset
```

Comment:
This is another change that needs to be reverted for the tests to pass. I think something is off between pre-commit we are using locally and pre-commit CircleCI is using

Reply:
Reverting these two files worked for me in the past:
```
src/home_robot/home_robot/perception/detection/lseg/modules/lsegmentation_module.py
src/home_robot_hw/home_robot_hw/nodes/contact_graspnet_server.py
```

Reply:
Yep, I think that's it. I tried `pre-commit clean/gc/uninstall` and then reinstallating the venvs but I see the same linting and formatting behavior which does not match the expected output by the CI 😭

Reply:
I ended up uninstalling pre-commit to make this work. If I checked out those two files pre-commit would again reformat them to break tests.